CHANGES.rst

```diff
@@ -1,6 +1,14 @@
 Changelog
 =========
 
+Version 1.1.3 [2021-11-12]
+--------------------------
+
+- `99b829 <https://github.com/openwisp/django-swappable-models/commit/99b829>`_: [feature] Add swappabled dependency pointing to __latest__
+- [deps] Verified support for python 3.9
+- [deps] Added support for Django 3.2 and Django 4.0a1
+- [deps] Dropped support for python<=3.6 and Django<2.2
```

Comment:
this seems missing:
https://github.com/openwisp/django-swappable-models/commit/1ac54fc06e91d41acb0b4a52f877e711b4c6a6e4